catalog/MDCCatalog/MDCCatalogComponentsController.swift

```diff
@@ -103,7 +103,11 @@ class MDCCatalogComponentsController: UICollectionViewController, MDCInkTouchCon
     let titleLabel = UILabel()
     titleLabel.text = self.title!.uppercased()
     titleLabel.textColor = UIColor(white: 1, alpha: 1)
-    titleLabel.font = UIFont(name: "RobotoMono-Regular", size: 14)
+    if #available(iOS 9.0, *) {
+        titleLabel.font = UIFont.monospacedDigitSystemFont(ofSize: 14, weight: UIFontWeightRegular)
+    } else {
+        titleLabel.font = UIFont.preferredFont(forTextStyle: .body)
```

Comment:
I've tried this, too, but I believe we're not moving to Dynamic Type just yet. 😿  Sorry!  I think it makes sense to use the same approach as you do in the Objective-C, get a system font with the correct size and apply the Monospace trait to the descriptor.